Orange/canvas/utils/settings.py

```diff
@@ -82,9 +79,9 @@ def qt_to_mapped_type(value):
         return value
 
 
-class QABCMeta(pyqtWrapperType, abc.ABCMeta):
+class QABCMeta(_QObjectType, abc.ABCMeta):
     def __init__(self, name, bases, attr_dict):
-        pyqtWrapperType.__init__(self, name, bases, attr_dict)
+        _QObjectType.__init__(self, name, bases, attr_dict)
         abc.ABCMeta.__init__(self, name, bases, attr_dict)
```

Comment:
https://docs.python.org/3.5/library/abc.html#abc.ABCMeta
It is the metaclass of the python's 'abstract' containers (`collections.Mapping`, `collection.Sequence`, ...), and needed here in order to properly subclass QObject and MutableMapping later on.